src/app/core/components/header/header.component.scss

```diff
@@ -0,0 +1,74 @@
+@import "../../styles/colors";
+
+.mat-sidenav-container{
+    height: 100%;
+    background-color: $dark;
+
+    .mat-sidenav {
+        padding: 20px;
+        background-color: $grey;
+        width: 20%;
+    }
+
+    .router-link {
+        background-color: blanchedalmond;
+        color: blue;
+    }
+
+    .mat-toolbar{ 
+        color: white;
+        height: 80px;
+        background-color: $grey;
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+
+        .sidebar {
+            display: flex;
+            align-items: center;
+
+            p {
+                margin: 0;
+            }
+        }
+
+        .nav {
+
+            .router-link-active {
+                color: $green;
+                border-top: 3px solid $green;
+
+                &:hover {
+                    background-color: $green;
+                    color: $white;
+                    border-top: 3px solid $white;
+                }
+            }
+
+            button {
+                border: none;
+                margin: 10px;
+                padding: 20px 10px;
+                color: $white;
+                font-size: 20px;
+                background-color: $grey;
+                outline: none;
+
+                &:hover {
+                    background-color: $green;
+                    color: $white;
+                }
+            }
+    
+            p {
+                margin: 0;
+                color: $white;
+            }
+        }
+           
+        // .user {
+
+        // }
+        
```

Comment:
Please remove these empty lines and commented code

Reply:
fixed